mdbenchmark/mdengines/namd.py

```diff
@@ -49,6 +52,8 @@ def prepare_benchmark(name, relative_path, *args, **kwargs):
 
     return name
 
+def prepare_multidir(multidir):
+    raise NotImplementedError
```

Comment:
Same as above

Reply:
The error here is actually not correct i think, because it will cause the namd generation to crash by default. Since there is no "multidir" string to be replaced in the namd-template, here an empty string is actually the correct retrun value.

Reply:
... or maybe better to return `None`
```suggestion
    return None
```